### PR TITLE
Fix to bug #112588

### PR DESCRIPTION
**Pull request by gougouasmi**
Fixes #112588

cc @svekars @brycebortree @carljparker

### CODE REVIEW

**Comment by kit1980** [comment]
@gougouasmi You need to add the email of this commit to your GitHub profile and sign the CLA.

**Comment by gougouasmi** [comment]
Hello @kit1980 ! Thank you for reaching out and sorry for the late reply.

I signed the CLA (I have the pdf), but then I looked into commit and emails on github (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) and I found out that:

1. The email associated with my commits was indeed incorrect as you pointed out. (git config user.email) 
2. On github.com, I set my email to be kept private, so I should have set the commit email to the noreply email github provided me with.

Sorry about this mess!

File: torch/_utils.py [hunk 1]
```diff
@@ -304,10 +308,6 @@ def _rebuild_sparse_tensor(layout, data):
     raise NotImplementedError(f"rebuilding sparse tensor for layout {layout}")
 
 
-def _rebuild_nested_tensor(buffer, sizes, strides, storage_offsets):
```
Comment (carljparker) [inline torch/_utils.py:307]: Were these lines _deleted_? I don't any corrected lines; it appears that these are gone.

File: torch/_utils.py [hunk 2]
```diff
@@ -837,13 +855,15 @@ def __get__(self, instance, owner=None):
 
 
 def classproperty(func):
+    """Class property descriptor."""
```
Comment (kit1980) [inline torch/_utils.py:858]: Placeholder?

**Pull request closed without merging**